CHANGELOG.md

```diff
@@ -2,414 +2,416 @@
 
 All notable changes to this project will be documented in this file. If a contribution does not have a mention next to it, [@geelen](https://github.com/geelen) or [@mxstbr](https://github.com/mxstbr) did it.
 
-*The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).*
```

Comment:
Can you remove your formatting from this file please? It’s be great to preserve its previous history and format

Reply:
sorry, didn't realise 🤔

Reply:
fixed